src/librustc/session/config.rs

```diff
@@ -1232,6 +1232,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
          Use with RUST_REGION_GRAPH=help for more info"),
     parse_only: bool = (false, parse_bool, [UNTRACKED],
         "parse only; do not compile, assemble, or link"),
+    dual_proc_macros: bool = (false, parse_bool, [TRACKED],
+        "load proc macros for both target and host, but only link to the target"),
```

Comment:
Sorry for being so late to see this, but can an issue be opened to track this "feature"?

Reply:
Opened https://github.com/rust-lang/rust/issues/64671